apps/dashboard/app/models/project.rb

```diff
@@ -281,7 +281,7 @@ def readme_path
 
   def zip_to_template
     # using ZipKit create a zip file called project.zip containiong the files named in file_names and save it in the project_dataroot
-    zip_file = "#{project_dataroot}/project.zip"
```

Comment:
I think this was correct before. Let it be a real file they can discover when they're browsing the files.